util/rtconf-test/inc/string.h

```diff
@@ -0,0 +1,169 @@
+#ifndef STRING_H
```

Comment:
@BartekLew is it really not available in coreboot?

Reply:
I think so, look at src/include/string.h
It's widely used.

Reply:
Then lets use string.h from `src/include/string.h`